CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file
 
+## [1.1.0] - 2020-04-16
+### Added
+- When a `Time` is passed in to a `QueryString` or `MatchCondition`, format it to use 3 decimal places (milliseconds) to guarantee it is compatible with millisecond timestamps stored in ES
+
```

Comment:
There should also be a `Changed` section indicating that, because of the `aggregate` changes, by default all times will now be stored with milliseconds precision.

Reply:
I'm not sure I agree. That change is strictly related to `aggregate`. If anything, we could mention that we bumped the version of `aggregate` to make use of the new version, which provides that change. 
Semantics, I know...

Reply:
> If anything, we could mention that we bumped the version of aggregate to make use of the new version, which provides that change.

That works 👍 As long as the change is pointed out in some way

Reply:
Ok. I buy that rationale.